setup.py

```diff
@@ -56,7 +56,7 @@ def load_requirements(file_list: Optional[Union[str, List[str]]] = None) -> List
 tsv_reqs = load_requirements("requirements/extra-csv.in")
 xlsx_reqs = load_requirements("requirements/extra-xlsx.in")
 
-all_doc_reqs = list(
+all_doc_reqs = sorted(
```

Comment:
Although I added no reproducibility test to this repo, the Pex repo now has one inspired directly by the prior `list(set(...))` construction. That test failed prior to Pex plumbing `PYTHONHASHSEED=0` to all process where distributions are built.